### PR TITLE
update manifest to make extension compatible with vs 2019

### LINKED ISSUE #468: Support for Visual Studio 2019 preview 1?
https://blogs.msdn.microsoft.com/visualstudio/2018/12/04/making-every-developer-more-productive-with-visual-studio-2019/

I tried installing Roslynator 17, both via the vsix and via VS2019's "Extensions and Updates" dialog, and it didn't show up. Roslynator has become an extension I literally cannot work without, so I basically can't use VS2019 until this extension is available there. 

 I appreciate that having to support an extension for an alpha product is unrealistic, so if you could just post instructions on how I can build the extension from source that would be fine.

I had a look at [source.extension.vsixmanifest](https://github.com/JosefPihrt/Roslynator/blob/master/src/VisualStudio/source.extension.vsixmanifest) which I assume is the file responsible for allowing extensions to be installed in the various VS versions, but it seems you already have 16.0/VS2019 in there, so not sure why it isn't working.

### PR DESCRIPTION
**Pull request by IanKemp**
closes #468

### CODE REVIEW

File: src/VisualStudio/source.extension.vsixmanifest [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
   <Metadata>
     <Identity Id="9289a8ab-1bb6-496b-9992-9f7ea27f66a8" Version="2.0.1" Language="en-US" Publisher="Josef Pihrt" />
-    <DisplayName>Roslynator 2017</DisplayName>
+    <DisplayName>Roslynator 2019</DisplayName>
```
[the remark this answers is not in the document]
Comment (drewnoakes) [reply src/VisualStudio/source.extension.vsixmanifest]: Why keep the name the same? Perhaps remove the year altogether.

[the remark this answers is not in the document]
Comment (IanKemp) [reply src/VisualStudio/source.extension.vsixmanifest]: TBH I've always wondered the same thing myself...

[the remark this answers is not in the document]
Comment (marcinsmialek) [reply src/VisualStudio/source.extension.vsixmanifest]: It appears like there was a split, and the old version (for VS2015) remained year-less, and the new one became 2017, which makes sense.
Even if the names get changed, I think it should be a separate commit.

[the remark this answers is not in the document]
Comment (marcinsmialek) [reply src/VisualStudio/source.extension.vsixmanifest]: @IanKemp do you think you could revert the name to the old one, and let @JosefPihrt decide about eventual name change later (in another commit)?

[the remark this answers is not in the document]
Comment (JosefPihrt) [reply src/VisualStudio/source.extension.vsixmanifest]: I took it back.

**Comment by JosefPihrt** [comment]
A new extension Roslynator 2019 will be introduced and it will be marked as "preview" as I expect bugs related to C# 8.0. It will reference newest Roslyn libraries (presumably 3.0).

Roslynator 2017 will have to stay at Roslyn 2.x to be compatible with Visual Studio 2017.

I would prefer to have one extension that would target VS2017 and VS2019, but it is not possible.

**Comment by JosefPihrt** [comment]
I'm going to continue with this in branch https://github.com/JosefPihrt/Roslynator/tree/2019.

**Pull request closed without merging**